pykusto/functions.py

```diff
@@ -2,6 +2,7 @@
 from pykusto.expressions import *
 from pykusto.expressions import _subexpr_to_kql, Column
 from pykusto.utils import KQL
+from expressions import DynamicExpression
```

Comment:
Incorrect import, supposed to be `from pykusto.expressions` (like above). If this worked for you then there is some problem with your project configuration

Reply:
I had similar issues in the past, I think #46  will help prevent those.